### PR TITLE
enable qe-approved label for compliance-operator and file-integrity-operator

### PR DESCRIPTION
**Pull request by JAORMX**

### CODE REVIEW

**Comment by JAORMX** [comment]
Pull-request updated, HEAD is now f9942e7abc8e07e855167c7f1e72125becf24519

**Review by petr-muller — APPROVED** [review]

File: core-services/prow/02_config/_labels.yaml [hunk 1]
```diff
@@ -653,6 +653,14 @@ repos:
         name: docs-approved
         target: both
         addedBy: label
+  openshift/compliance-operator:
```
Comment (petr-muller) [inline core-services/prow/02_config/_labels.yaml:656]: The labels need to be under `labels`, and the items are missing target (prs? issues? both?) and who adds the label

Comment (JAORMX) [reply core-services/prow/02_config/_labels.yaml:656]: Fixing, thanks!

Comment (petr-muller) [reply core-services/prow/02_config/_labels.yaml:656]: Looks like you added the missing fields to only one label, not both. Probalby just an omission?

Comment (JAORMX) [reply core-services/prow/02_config/_labels.yaml:656]: just me not being able to copy/paste appropriately...

**Review by petr-muller — APPROVED** [review]

**Pull request merged**